src/features/minimap/MinimapOptions.tsx

```diff
@@ -8,14 +8,18 @@ import React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Button } from '@nordicsemiconductor/pc-nrfconnect-shared';
 
+import { isRealTimePane as isRealTimePaneSelector } from '../../utils/panes';
 import {
     setShowMinimapAction,
     showMinimap as getShowMinimap,
 } from './minimapSlice';
 
 export default () => {
-    const showMinimap = useSelector(getShowMinimap);
     const dispatch = useDispatch();
+    const showMinimap = useSelector(getShowMinimap);
+    const isRealTimePane = useSelector(isRealTimePaneSelector);
+
+    if (isRealTimePane) return null;
```

Comment:
For other options this is solved in `SidePanel.tsx`, so maybe also do this there instead of here: https://github.com/NordicSemiconductor/pc-nrfconnect-ppk/blob/a3536fd622bd2a03b4f67e8b8451b03a38988a37/src/components/SidePanel/SidePanel.tsx#L84-L85